DFKPS/DFKPS-IOC-01App/Db/DFKPS_8800_status.db

```diff
@@ -344,8 +344,8 @@ record(ao, "$(P)SIM:RAW"){
     field(DTYP, "Soft Channel")
     field(EGU, "A")
 }
-alias("$(P)SIM:CURR", "$(P)SIM:RAW:SP")
-alias("$(P)SIM:CURR", "$(P)SIM:RAW")
+alias("$(P)SIM:RAW", "$(P)SIM:RAW:SP")
+alias("$(P)SIM:RAW", "$(P)SIM:CURR")
```

Comment:
This is still conflicting with something I think there is a SIM:CURR in st-common.db which might be the trouble. I get the following error when I run it:

```
dbLoadRecords("C:/Instrument/Apps/EPICS/ioc/master/DFKPS/Db/DFKPS_8800_status.db
", "device=NDW1407:HGV27692:DFKPS_01, P=NDW1407:HGV27692:DFKPS_01:, port=L0, SP_
PINI=NO, FRI=1")
Can't create alias "NDW1407:HGV27692:DFKPS_01:SIM:RAW:SP" referring to "NDW1407:
HGV27692:DFKPS_01:SIM:RAW"
Error at or before ")" in file "C:/Instrument/Apps/EPICS/ioc/master/DFKPS/Db/DFK
PS_8800_status.db" line 347
Can't create alias "NDW1407:HGV27692:DFKPS_01:SIM:CURR" referring to "NDW1407:HG
V27692:DFKPS_01:SIM:RAW"
Error at or before ")" in file "C:/Instrument/Apps/EPICS/ioc/master/DFKPS/Db/DFK
PS_8800_status.db" line 348
```

and similar for 8000.